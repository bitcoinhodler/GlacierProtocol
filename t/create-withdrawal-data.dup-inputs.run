#!/bin/bash
set -e

$GLACIERSCRIPT --regtest=$1 create-withdrawal-data << INPUT
y
y
y
y
y
y
2N93du8YobdgsHyu3qgBvSyhGUT52utMNeA
522103d14ddcfb6817f5579695bbb3eb3e185887bf2942b031e6f716343b8fe7e9e8e221028fcd46f8614b2cbf318096968242a1e22bcfb6d8f2b6dc939c8c27c347b2937b210315acb550120f4cdcb460d5c49080ab508ca4ccd8dedecac22feeac8cd017d4c121022b063ee2f22f9e1982c140fe778672c683111a796dcccdbd47bb65e3d608a98354ae
myP4xdJNwAW9iMakvCjnozg814ewgn8apd
2
02000000015f709f35c24c4a42a452904074cdbf8b2d71df40f3cbe1666140d0348e4f29c2010000006a473044022062537d72e67f97ad493d978d3106f17cac630308fc582580c1727178898015cf022051b1d59971563d12abeb9709eaa979fca50fd06cd6ad61a9f9bf72c5a5716a18012102e54bef15356decb04e874f025994497ff323c3fad74d6a9d60dc070640fc8f30ffffffff02d3f55b05000000001976a914023e4b8e3e3b3432beecf3ebe95876690dbfbd1f88ac809698000000000017a914ad50fb2489353008e75b0193011ff1f4b76f798c8700000000
02000000015f709f35c24c4a42a452904074cdbf8b2d71df40f3cbe1666140d0348e4f29c2010000006a473044022062537d72e67f97ad493d978d3106f17cac630308fc582580c1727178898015cf022051b1d59971563d12abeb9709eaa979fca50fd06cd6ad61a9f9bf72c5a5716a18012102e54bef15356decb04e874f025994497ff323c3fad74d6a9d60dc070640fc8f30ffffffff02d3f55b05000000001976a914023e4b8e3e3b3432beecf3ebe95876690dbfbd1f88ac809698000000000017a914ad50fb2489353008e75b0193011ff1f4b76f798c8700000000
INPUT
