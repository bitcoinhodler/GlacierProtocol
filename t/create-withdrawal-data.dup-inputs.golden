Are you running this on a computer WITHOUT a network connection of any kind? (y/n)?Have the wireless cards in this computer been physically removed? (y/n)?Are you running on battery power? (y/n)?Are you running on an operating system booted from a USB drive? (y/n)?Is your screen hidden from view of windows, cameras, and other people? (y/n)?Are smartphones and all other nearby devices turned off and in a Faraday bag? (y/n)?
You will need to enter several pieces of information to create a withdrawal transaction.


*** PLEASE BE SURE TO ENTER THE CORRECT DESTINATION ADDRESS ***


Source cold storage address: 
Redemption script for source cold storage address: 
Destination address: 
How many unspent transactions will you be using for this withdrawal? 
Please paste raw transaction #1 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):

Please paste raw transaction #2 (hexadecimal format) with unspent outputs at the source address
OR
input a filename located in the current directory which contains the raw transaction data
(If the transaction data is over ~4000 characters long, you _must_ use a file.):
ERROR: duplicated input transactions. Exiting...
